src/components/scores.jsx

```diff
@@ -354,14 +349,18 @@ const Scores = ({ inst_id, play_id, single_id, send_token, isEmbedded, isPreview
 					href += `?token=${window.LAUNCH_TOKEN}`
 				}
 				setAttemptsLeft(attemptsLeft)
-				setHidePlayAgain(false)
-				setWidget({
-					...widget,
-					href: href
+				
+				setAttributes({
+					...attributes,
+					href: href,
+					hidePlayAgain: false
 				})
 			} else {
 				// if there are no attempts left, hide play again
-				setHidePlayAgain(true)
+				setAttributes({
+					...attributes,
+					hidePlayAgain: true
+				})
 			}
 		}
 	}, [attemptsLeft])
```

Comment:
Thanks for the sanity check. This got rearranged so that `setAttemptsLeft` is only assigned when the `apiGetWidgetInstanceScores` query resolves, which itself is predicated on `instance` existing.